docs/100_producten/100_borgstelling/borgstelling-aanvragen.feature

```diff
@@ -90,7 +90,7 @@ Functionaliteit: Borgstelling aanvragen
     En het Schuldenknooppunt bericht is verwerkt
     Dan is de status van de borgstelling "afgegeven"
     En is "het contract" gearchiveerd
-    En is er een verkoop geregistreerd van € "20.0"
+    En is er een verkoop geregistreerd van € "49.99"
```

Comment:
```suggestion
    En is er een verkoop geregistreerd van € 20,00
```
In het geval van een overname moet niet de bruto kredietsom maar het uitstaand saldo gebruikt worden voor de premie berekening. Dus 1% van 2000.